### PR TITLE
Add a Dockerfile for the application so that it can be run as an ECS …

### PR DESCRIPTION
**Pull request by aaron-collier**
…task

## Why was this change made?

This is a simple dockerization of this application so that it can be run within our ECS Cluster as a task when triggered from Airflow.

## How was this change tested?

Manually verified the image works and runs the executable properly.

## Which documentation and/or configurations were updated?

README

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM ruby:2.7.2
+
+WORKDIR /app
+# # Operating system dependencies
```
Comment (justinlittman) [inline Dockerfile:4]: Extra `#`

**Review by justinlittman — APPROVED** [review]
See minor comment.

Do you want to build via Circle?

**Comment by justinlittman** [comment]
@aaron-collier Can this PR be closed?

**Comment by aaron-collier** [comment]
Yeah - apologies, went a different direction and didn't circle back to this. closed.

**Pull request closed without merging**